test/openAPI/features/support/helpers/helpers.js

```diff
@@ -45,6 +45,49 @@ module.exports = {
       },
     ],
   },
+  //applicant_applications_fileId
+  applicationFileEndpoint: 'applications/{fileId}',
+  applicationFileResponseSchema: {
+    type: 'object',
+    properties: {
+      fileId: { type: 'string' },
+      registered: { type: 'string' },
```

Comment:
In $date-time format, example: 2000-10-23T00:00:00.000Z

Reply:
What do you mean? Should be there sone other type?
updated the rest just have no idea about this one